application/actions/meetings.go

```diff
@@ -127,15 +128,21 @@ func meetingsUpdate(c buffalo.Context) error {
 		return reportError(c, err)
 	}
 
+	if !cUser.CanUpdateMeeting(meeting) {
+		err := fmt.Errorf("user %s is not authorized to update meeting %s",
```

Comment:
Good thinking to add IDs to the error log. Rather than add the user and meeting IDs to the message, it might be better to add them to "extras". That way the rollbar message is identical for all occurrences, so would be grouped better. I think it may recognize the message with IDs as being similar, but I think it's still better to use extras. Also, fwiw, I think the integer ID is fine in an error log. The UUID is mostly for end-user visibility.

Reply:
:+1: